mlserver/metrics/server.py

```diff
@@ -37,6 +35,14 @@ async def start(self):
         await self._server.serve()
 
     def _get_config(self):
+        kwargs = {}
+
+        if self._settings._custom_metrics_server_settings:
+            logger.warning(
+                "REST custom configuration is out of support. Use as your own risk"
+            )
+            kwargs.update(self._settings._custom_metrics_server_settings)
+
         kwargs = {
```

Comment:
Shouldn't the `kwargs` be merged instead of reassigned?

Reply:
You're right. Fixed